app/controllers/providers/applicant_details_controller.rb

```diff
@@ -19,7 +19,7 @@ def applicant
 
     def form_params
```

Comment:
Do we use this controller?  🤔

Reply:
this was a confusing one for me too but yes I am pretty sure we do. it handles the edit/update aspect of applicant details while the `app/controllers/providers/applicants_controller.rb` handles the new/create.

Reply:
Cool, I mean - not cool, but thanks for investigating!